src/yogita/Exam1.java

```diff
@@ -0,0 +1,27 @@
+package yogita;
+class Exam1{
+		
+   static String maximumLength(String[] arr) { 
+	   String  max = arr[0];
+	   for(int index=0;index<arr.length;index++)
+	      if(max.length()<arr[index].length())
+	 	    max = arr[index];
+	return max;
+	}
+	
+   void even(String name) {
+	   for(int index=0;index<name.length();index++)
+		  if(index % 2 == 0)
```

Comment:
look at the requirement. we want to print even character , index starts from 0. Odd index represents even characters of the string. look at the expected output. 

input : Technocredits
output : ehordt
You should write if(index%2 != 0)